src/app/item-details/item-details.component.ts

```diff
@@ -1,15 +1,26 @@
-import { Component, OnInit } from '@angular/core';
+import { DaysLeftCountedPipe } from './../pipes/daysLeftCountedPipe/days-left-counted.pipe';
+import { ActionItem } from './../services/tasksService/tasks.service';
+import { Component, Input, ChangeDetectionStrategy, OnChanges } from '@angular/core';
 
 @Component({
   selector: 'app-item-details',
   templateUrl: './item-details.component.html',
-  styleUrls: ['./item-details.component.css']
+  styleUrls: ['./item-details.component.css'],
+  providers: [DaysLeftCountedPipe],
```

Comment:
It would be better to provide this pipe in some module rather than in the component.

Reply:
This one is still not addressed.